usage/ui.md

```diff
@@ -32,6 +32,24 @@ mix.webpackConfig({
 ...
 ```
 
+> Be sure to set `render_on_redirect => true` in the Livewire configuration file `config/livewire.php`.
```

Comment:
Are there any differences in livewire configuration files between projects? If not we can keep a default livewire config in our template repository as this is a one time setting that has to be changed.